lib/wsdl.js

```diff
@@ -1558,7 +1558,11 @@ WSDL.prototype.objectToXML = function(obj, name, namespace, xmlns, first, xmlnsA
       }
 
       if (!Array.isArray(child)) {
-        parts.push(['<', ns, name, attr, xmlnsAttrib, '>'].join(''));
+        var insNull = '';
+        if(typeof child === 'object' && child === null) {
```

Comment:
Thanks a lot for your understanding - I hope you don't get me wrong - I'm not nagging, I'm just up for learning as much as I can from others :smiley: